app/__init__.py

```diff
@@ -25,6 +25,14 @@ def create_app(config):
     app.config['JWT_SECRET_KEY'] = app.config['SECRET_KEY']
     jwt = JWTManager(app)
 
+    # Return a more interpretable message if auth fails due to an invalid token
+    @jwt.invalid_token_loader
+    def informative_invalid_token_callback(invalid_token):
+        return jsonify({
```

Comment:
The client doesn't try to parse JSON out of error messages (though I can do it if needed) and it already has the status code, so maybe we should just return the message instead of JSON to keep the error more readable.